src/main/resources/bucketeer.yaml

```diff
@@ -20,6 +20,50 @@ paths:
       responses:
         '200':
           description: OK
+  /batch/jobs:
+    get:
+      summary: Jobs in Progress
+      description: A listing of batch jobs currently in practice
```

Comment:
It's a tiny thing, but I think "practice" may not be the word you want? Maybe "progress"?